src/Database/Model.php

```diff
@@ -672,6 +672,17 @@ protected function performDeleteOnRelations()
     //
     // Adders
     //
+    
+    /**
+     * Add attribute casts for the model.
+     * 
+     * @param  array $attributes
+     * @return void
+     */
+    public function addCasts($attributes)
```

Comment:
Could you make this match the other setters? I.e. pass multiple strings or an array of strings?

Reply:
No, that API doesn't make sense in this context. Unlike the other adders, a model's `casts` value is an associative array.

Reply:
Fair enough